server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java

```diff
@@ -45,7 +45,7 @@ public void setUp() {
     @Test
     public void PropertyRepository_Save_ReturnSavedPropertyById() {
         // Assert
-        Optional<Property> savedProperty = propertyRepository.findById(id);
+        Optional<PropertyDetails> savedProperty = propertyRepository.findById(id);
```

Comment:
This is my unit test please don't change it

Reply:
Fixed it